BitwardenShared/UI/Autofill/Utilities/Fido2UserInterfaceHelper.swift

```diff
@@ -0,0 +1,74 @@
+import BitwardenSdk
+import Combine
+
+/// A helper to extend `Fido2UserInterface` protocol capabilities for Fido2 flows
+/// depending on user interaction.
+protocol Fido2UserInterfaceHelper: Fido2UserInterface {
+    /// The `BitwardenSdk.Fido2CredentialNewView` the SDK provides while in FIdo2 creation flow.
+    var fido2CredentialNewView: BitwardenSdk.Fido2CredentialNewView? { get }
+
+    /// Sets the selected cipher as a result for credential for Fido2 creation.
+    /// - Parameter result: The result of picking a cipher with the cipher or the error.
+    func pickedCredentialForCreation(result: Result<CheckUserAndPickCredentialForCreationResult, Error>)
+
+    /// Sets up the delegate to use on Fido2 user verification flows.
+    /// - Parameter fido2UserVerificationMediatorDelegate: The delegate to use.
+    func setupDelegate(fido2UserVerificationMediatorDelegate: Fido2UserVerificationMediatorDelegate)
+}
+
+/// Default implemenation of `Fido2UserInterfaceHelper`.
+class DefaultFido2UserInterfaceHelper: Fido2UserInterfaceHelper {
+    /// Mediator which manages user verification on Fido2 flows.
+    private var fido2UserVerificationMediator: Fido2UserVerificationMediator
+
+    /// Continuation when picking a credential for creation.
+    var credentialForCreationContinuation: CheckedContinuation<CheckUserAndPickCredentialForCreationResult, Error>?
+
+    private(set) var fido2CredentialNewView: BitwardenSdk.Fido2CredentialNewView?
```

Comment:
⛏️ Could you add docs for this?

Reply:
The docs for this is already in the protocol, I don't see that being repeated in the implementation in other places. Or should I add it nonetheless because it's a property instead of a function?

Reply:
Ah, that's fine then we usually don't document protocol methods/variables.